### PR TITLE
Minor test improvements

### PR DESCRIPTION
**Pull request by maringuu**

### CODE REVIEW

**Comment by maringuu** [comment]
https://github.com/fkie-cad/FACT_core/actions/runs/3793900286

**Comment by maringuu** [comment]
https://github.com/fkie-cad/FACT_core/actions/runs/3799434870

File: src/conftest.py [hunk 1]
```diff
@@ -11,10 +13,19 @@
 import config
 from analysis.PluginBase import AnalysisBasePlugin
 from config import Config
-from test.common_helper import CommonDatabaseMock, create_docker_mount_base_dir
+from test.common_helper import CommonDatabaseMock
 from test.conftest import merge_markers
 
 
+def _create_docker_mount_base_dir() -> str:
+    dir = tempfile.mkdtemp(prefix='fact-docker-mount-base-dir')
```
Comment (jstucke) [inline src/conftest.py:21]: `dir` is a python builtin and should not be used

```suggestion
    tmp_dir = tempfile.mkdtemp(prefix='fact-docker-mount-base-dir')
```

File: src/conftest.py [hunk 2]
```diff
@@ -11,10 +13,19 @@
...
+    docker_gid = grp.getgrnam('docker').gr_gid
+    os.chown(dir, -1, docker_gid)
+    os.chmod(dir, 0o770)
```
Comment (jstucke) [inline src/conftest.py:24]: This part seems to be mostly the same as what happens at the beginning of the `main()` method in `src/start_fact_backend.py`. Would it make sense to put it in a helper function (in fileSystem.py or docker.py maybe)?

**Review by jstucke — CHANGES_REQUESTED** [review]
Looks good apart from some small stuff

**Review by jstucke — APPROVED** [review]

**Comment by maringuu** [comment]
https://github.com/fkie-cad/FACT_core/actions/runs/3883411072

**Pull request merged**